src/test/common/fdc3.app-channels.ts

```diff
@@ -137,12 +137,18 @@ export function createAppChannelTests(cc: ChannelControl<any,any>, documentation
       it(acTestId8, async () => {
         const errorMessage = `\r\nSteps to reproduce:\r\n- App A retrieves an app channel\r\n- App A switches to a different app channel\r\n- App A adds a context listener of type fdc3.instrument\r\n- App B retrieves the first channel that A retrieved\r\n- App B broadcasts a context of type fdc3.instrument${documentation}`;
 
-        let testChannel = await cc.createTestChannel()
+        let testChannel1 = await cc.createTestChannel()
+        let receivedContext = false;
+
         const resolveExecutionCompleteListener = cc.initCompleteListener(acTestId8)
-        testChannel = await cc.createTestChannel("a-different-test-channel");
-        await cc.setupAndValidateListener1(testChannel, "unexpected-context", errorMessage, () => { /*noop*/ })
-        await cc.openChannelApp(acTestId8, undefined, APP_CHANNEL_AND_BROADCAST)
+        let testChannel2 = await cc.createTestChannel("a-different-test-channel");
+        await cc.setupAndValidateListener1(testChannel1, "fdc3.instrument", errorMessage, () => { receivedContext = true })
```

Comment:
based on suggested change to util function
```suggestion
        await cc.setupAndValidateListener1(testChannel1, "fdc3.instrument", "fdc3.instrument", errorMessage, () => { receivedContext = true })
```